symphonia-codec-aac/src/adts.rs

```diff
@@ -428,34 +428,48 @@ fn approximate_frame_count(mut source: &mut MediaSourceStream<'_>) -> Result<Opt
     else {
         // The number of points to sample within the stream.
         const NUM_SAMPLE_POINTS: u64 = 4;
+        const NUM_FRAMES: u32 = 100;
```

Comment:
A slightly more descriptive name.

```suggestion
        const NUM_FRAMES_PER_SAMPLE: u32 = 100;
```